### PR TITLE
Add explanation about creating a bigger Hash Onion

### LINKED ISSUE #1931: Add explanations about hashonions
### Motivation
- Explain how to generate bigger hashonions offline
- Review existing commands regarding hashonion (address parameter needed?)

```
lisk-core hash-onion -o ./hash_onion.json
// edit hash_onion.json to add "address": "xxx"
lisk-core endpoint:invoke random_setHashOnion -f ./hash_onion.json
```

#### Affected branch / URL
`main` /
https://liskhq.github.io/lisk-docs/run-blockchain/become-a-validator.html#set-the-hash-onion

#### Page location
`run-blockchain/become-a-validator.adoc#set-the-hash-onion`

### Additional resources
https://github.com/Gr33nDrag0n69/LiskCore4Tools/blob/main/MD/Core4_NpmInstall_MaxHashComputation_Patch.md#how-official-undocumented-method
https://github.com/LiskHQ/lisk-sdk/issues/9174

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1931

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc [hunk 1]
```diff
@@ -383,6 +383,70 @@ That will return the validator keys that are currently imported on the node:
 ]}
 ----
 
+== Generating the hash onion
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc]: Please don't create a new section for this, it creates redundancy and inconsistency with the next section.

Add the info as an alternative und "Set the hash onion", maybe as info box or if it's too big for that create tabs and explain both options how to create hashonions (default way vs custom way)

Also, I wouldn't explain how to create a bash script step by step here - either provide the commands directly for easy copy-paste, or provide the complete bash script and show a usage example of the script

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc [hunk 2]
```diff
@@ -386,25 +387,58 @@ That will return the validator keys that are currently imported on the node:
...
+./bin/run endpoint:invoke random_setHashOnion '{"address":"lskqaxxmj78frvgpjgwvf4yqjjkcrr9yhn2sxxwm3"}'
 ----
 --
-Sidechain node::
+Setting a custom hash onion::
 +
 --
+
+Each validator generating a block has to set the hash onion value.
+For every block that a validator generates, a hash onion is consumed.
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/become-a-validator.adoc]:
```suggestion
```

would remove this here, this is already covered in the intro text of the section

**Review by Tschakki — APPROVED** [review]
LGTM

**Review by ishantiw — APPROVED** [review]
LGTM

**Pull request merged**